code-challenges.test.js

```diff
@@ -20,8 +20,38 @@ const fibLength1 = 6
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// describe("fiboSeq", () => {
+  // it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.", () => {
+  //   expect(fiboSeq(fibLength1)).toEqual("[1, 1, 2, 3, 5, 8]")
+  //   expect(fiboSeq(fibLength2)).toEqual("[1, 1, 2, 3, 5, 8, 13, 21, 34, 55]")
```

Comment:
Nice setup for the test! Just keep in mind what we expect it to equal. Do we need the "" around the array?